brainiak/fcma/voxelselector.py

```diff
@@ -28,6 +28,7 @@
 from . import fcma_extension
 from . import cython_blas as blas
 import logging
+import pathos.multiprocessing
```

Comment:
According to @mjanderson09 , the standalone multiprocessing can only serialize methods at the root level, and the Pathos version is more flexible so that we can define a lambda function and use it in the `map` method.